keyboards/handwired/yamsek/atmega32u4/config.h

```diff
@@ -0,0 +1,8 @@
+// Copyright 2022 Johannes Schneider (@JohSchneider)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+
+#define RGB_DI_PIN B6
+#define RGBLED_NUM 3
+
```

Comment:
As you are targeting `develop`, you need to move these settings to `info.json` under the updated names (`ws2812.pin` and `rgblight.led_count`).
```suggestion
```